tests/uri_router_test.py

```diff
@@ -9,6 +9,24 @@
 pytest.importorskip('srwl_bl')
 
 
+def test_error_for_bots():
+    from pykern.pkunit import pkeq, pkexcept, pkre
+    from pykern.pkcollections import PKDict
+    from sirepo import srunit
+
+    fc = srunit.flask_client()
+    fc.sr_login_as_guest()
+
+    uri = '/get-application-data'
+    d = PKDict(simulationType='srw', method='NO SUCH METHOD')
+
+    r = fc.post(uri, json=d)
+    pkeq(200, r.status_code)
```

Comment:
I wouldn't expect this to succeed. Document it by perhaps checking the return result redirect to /#not-found or whatever we expect to happen.

Reply:
f04f83c

I added a known "good" user agent to this test, and a check for the error page in the result data